README.md

```diff
@@ -65,6 +65,7 @@ ReactDOM.render(
 | title                  | String\|React.Element          |           | Title of the dialog                                                             |         |
 | footer                 | React.Element                  |           | footer of the dialog                                                            |         |
 | closable               | Boolean                        | true      | whether show close button                                                       |         |
+| closeBtnIsDisabled     | Boolean                        | false     | whether close button is disabled                                                |         |
```

Comment:
I think a better name would be disableCloseBtn

Reply:
done